toolchains/iOS.framework.cmake

```diff
@@ -3,7 +3,7 @@ include(${CMAKE_SOURCE_DIR}/toolchains/iOS.toolchain.cmake)
 message(STATUS "Build for iOS archs " ${CMAKE_OSX_ARCHITECTURES})
 
 set(FRAMEWORK_NAME TangramMap)
-set(FRAMEWORK_VERSION "1.0")
```

Comment:
I don't believe Cocoapods has the same concept of SNAPSHOT artifacts as Maven/Gradle. Seems like the closest analog would be pointing the podspec at a specific commit hash?